lib/jsonapi_parameters/jsonapi_schema.json

```diff
@@ -0,0 +1,397 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
there is draft -07

Reply:
When this PR was created, I doubt 07 was released.

Even now 07 is not edge: http://json-schema.org/draft/2019-09/release-notes.html

I believe we should resolve the subject of version update (and whether it can be minor upgrade) in our library and then we can think about moving on to next version of schema.

Reply:
https://json-schema.org/draft/2019-09/schema
that's right. Therefore the draft was upgraded also in article linked in repo 👍

Reply:
https://github.com/davishmcclurg/json_schemer/issues/44
is supporting max draft-07 currently according to readme

Reply:
>Therefore the draft was upgraded also in article linked in repo 👍

I have no clue which article or link you are talking about.

Reply:
https://github.com/davishmcclurg/json_schemer

Reply:
oh wrong link: https://polythematik.de/2020/02/17/ruby-json-schema/